app/views/layouts/_logo.html.erb

```diff
@@ -1,6 +1,6 @@
 <a href="/" class="site-logo" aria-label="<%= t("views.main.aria_home") %>">
   <% if Settings::General.logo_svg.present? %>
-    <%= logo_svg %>
+    <img src="<%= logo_svg %>" alt="<%= community_name %>" style="object-fit: contain;max-width: 300px;max-height: 40px;" />
```

Comment:
I modified this to an `<img />` tag so that we can except any image format including SVG. We'll need to know the proper dimensions so this doesn't leak out of the layout. For now I went with `object-fit: contain;max-width: 300px;max-height: 40px;`.

Reply:
@nickytonline the svg upload is now working, however, it's not showing up visually because we need to add a width/height to the image since the svg is showing up as 0px.  Can you help me to address this please?